src/WinRT.Runtime/ComWrappersSupport.cs

```diff
@@ -113,7 +113,10 @@ internal unsafe static bool IsFreeThreaded(IObjectReference objRef)
 
         public static IObjectReference GetObjectReferenceForInterface(IntPtr externalComObject)
         {
-            return GetObjectReferenceForInterface<IUnknownVftbl>(externalComObject, IID.IID_IUnknown);
+            // Here the ptr itself might not point to IUnknown, but we are using IUnknown for the purposes of getting
+            // an agile reference if needed.  Due to that and to keep back compat, making sure to not trigger a QI
+            // In addition, the ptr is already pointing to the correct interface which the IObjectReference is expected for.
+            return GetObjectReferenceForInterface<IUnknownVftbl>(externalComObject, IID.IID_IUnknown, false);

```

Comment:
Nit: I'd always explicitly use `requireQI: true` (or `false`) at these callsites just for extra clarity.

Reply:
Addressed.